2017-2018/GameEngine/comm/pacmanState.proto

```diff
@@ -7,7 +7,7 @@ message PacmanState {
   message AgentState {
     required int32 x = 1;
     required int32 y = 2;
-    required Direction direction = 3;
+    optional Direction direction = 3;
```

Comment:
If I remember correctly, Google says to never make a required field optional later. In this case, since we haven't published our code yet, it's okay, but definitely something to be avoided. This is one reason they give to avoid required in general.

Reply:
I think it was the other way around, because that way you would break backwards compatibility. Regardless, we haven't published the code, so it's fine